test/app/views/application/content_for.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render ContentForComponent do |comp| %>
+  <p>Have a great day.</p>
+  <% comp.content_for(:title) do %>
+    <h1>Hi!</h1>
```

Comment:
For sanity's sake, might we evaluate some Ruby inside these blocks? Even something like:

```suggestion
    <h1>Hi! <%= 1+ 2 %></h1>
```

Reply:
sure